types/dom-chromium-installation-events/index.d.ts

```diff
@@ -0,0 +1,30 @@
+// Type definitions for non-npm package dom-chromium-installation-events 101.0
+// Project: https://developer.mozilla.org/en-US/docs/Web/API/BeforeInstallPromptEvent
+// Definitions by: Sergey Kozlov <https://github.com/dartess>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export type AppBannerPromptOutcome = 'accepted' | 'dismissed';
+
+export interface PromptResponseObject {
+    readonly outcome: AppBannerPromptOutcome;
+    readonly platform: string;
+}
+
+declare global {
+    class BeforeInstallPromptEvent extends Event {
+        constructor(type: string, eventInitDict?: EventInit);
```

Comment:
this one is documented as parameterless:
https://developer.mozilla.org/en-US/docs/Web/API/BeforeInstallPromptEvent#constructor

Reply:
assuming convention for custom events